### DISCUSSION TITLE
Anzeigefehler

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#1699] [🇩🇪 Fragen&Antworten] [answered]
RaspberryMatic-3.61.7.20220205-f1c6c2-rpi3 , Englisch ist eingestell
![22](https://user-images.githubusercontent.com/81189856/152658822-207552e5-072a-4240-9a74-6e36b1964fa0.jpg)
t

**Comment by jp112sdl** [comment, marked as answer]
Ah, da ist ein Bug in der `en`-Übersetzung.
2 Fehler in einer Zeile... 👀 
https://github.com/jens-maus/RaspberryMatic/blob/0d5cb45554feb14f444584d5e5bf288e65ac411a/buildroot-external/patches/occu/0164-WebUI-Add-HMIPServerLogConfiguration/occu/WebUI/www/webui/js/lang/en/translate.lang.js#L494

muss lauten:
`"dialogSettingsCMLblLogHomematicIP" : HMIdentifier.en.HomematicIP,`

**Reply by jp112sdl** [reply]
@jens-maus Kannst du das a) selbst schnell ändern oder b) die Disukussion in ein Issue umwandeln?

**Reply by jens-maus** [reply]
Erledigt in 70e014366a43614374a36d47d6e21de1e992cc92
